homeassistant/components/coolmaster/__init__.py

```diff
@@ -22,7 +22,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     host = entry.data[CONF_HOST]
     port = entry.data[CONF_PORT]
     coolmaster = CoolMasterNet(
-        host, port, swing_support=entry.data.get(CONF_SWING_SUPPORT, False)
+        host,
+        port,
+        read_timeout=5 if entry.data.get(CONF_SWING_SUPPORT) else 1,
```

Comment:
Let's pull this if/else statement out of here (not in-line) and add a comment to the code that explains the reason why we do this.

Reply:
Done.